Makefile

```diff
@@ -62,6 +62,9 @@ test-short:
 format:
 	tox -e format
 
+lint:
```

Comment:
we have `test` in the makefile, what do you think about renaming that one?

Reply:
yea, perhaps removing it for good is a good idea. it's pretty misleading, to me at least

Reply:
how about now?